src/interface/misc.rs

```diff
@@ -130,3 +150,9 @@ impl AdvisoryLock {
         } else {false}
     }
 }
+
```

Comment:
this is re-appearing since networking was created before I fixed this so it should be removed for correctness. OTOH I do probably need a solution here at some point.

Reply:
What in particular are you referring to?

Reply:
The clone implementation for AdvLock that creates a new lock instead of copying/refcounting. This was implemented for a short period of time before we realized it was wrong and switched to refcounting. This seems to be a remnant from that.